src/pipe_execute.c

```diff
@@ -41,13 +41,16 @@ void	execute_pipe(int *flag, int *fds, t_command *cmd, int pos)
 	int		origin_argc;
 	char	**origin_argv;
 
+	set_fd(cmd);
 	signal(SIGINT, newline_hook);
 	if (!fork())
 	{
 		if (!flag[0])
 			dup2(fds[0], STDIN_FILENO);
 		if (!flag[1])
 			dup2(fds[3], STDOUT_FILENO);
+		else
+			dup2(cmd->fd, STDOUT_FILENO);
 		i = 0;
 		while (i < 4)
 			close(fds[i++]);
```

Comment:
We can move this portion of code into a external function iterating over i then returning to pass on norminette? Something like close_fds... Can you provide an example of expected behavior and bug found, so that I can test?

Reply:
I moved signal elsewhere just to pass norminette. That's the power of gambiarra

Reply:
![image](https://user-images.githubusercontent.com/11761170/172029402-9f3feaa3-64f7-4dda-bf84-7342f8f3e321.png)